eventmesh-openschema/eventmesh-openschema-registry/build.gradle

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+plugins {
+    id 'org.springframework.boot' version '2.5.4'
+    id 'io.spring.dependency-management'
+    id 'java'
+}
+
+group 'org.apache.eventmesh'
+version '1.2.0-SNAPSHOT'
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-data-jdbc'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-jdbc'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework:spring-beans:5.1.8.RELEASE'
+    implementation 'org.mybatis.spring.boot:mybatis-spring-boot-starter:2.2.0'
```

Comment:
Why did you choose to add both jpa and mybatis? It seems you can just choose one of them.
I prefer mybatis.

Reply:
I was not familiar with the difference between JPA and mybatis.  So I compared both of them and choose JPA due to its automatic generation of many query languages.

Next, I will try to modify the JPA part to mybatis since mybatis seems proposed and better maintainable by the community.

Reply:
Use JPA is OK, you can remove the `mybatis`.

Reply:
> Use JPA is OK, you can remove the `mybatis`.

Ya, Using Spring JPA should be fine. :)